lessons/modelevaluation.ipynb

```diff
@@ -0,0 +1,786 @@
+    "Notice that if your two models are equally probable _a priori_, then \n",
+    "\n",
+    "$\\frac{P(H_2)}{P(H_1} = 1$ such that $\\frac{P(H_2|D)}{P(H_1|D)} = R$\n",
+    "\n",
+    "This assumption is often not always easy to justify."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "A more practical way to interpret the Bayes factor is to note that it updates the model prior ratio into a posterior one. This means that:\n",
+    "\n",
+    "  * If you think that, despite having seen the data, your two models are *still equally probable,*\n",
+    "  \n",
+    "  * then the evidence ratio in favor of $H_2$ is _the odds that you would have had to have been willing to take against $H_2$, before seeing the data._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "In our linear model fit example, we can compute the evidence for the linear and quadratic models, and form the odds ratio $R$.\n",
+    "\n",
+    "```\n",
+    "log Evidence for Straight Line Model: -157.2\n",
+    "log Evidence for Quadratic Model: -120.7\n",
+    "Evidence ratio in favour of the Quadratic Model:\n",
+    "  7e15 to 1\n",
+    "```\n",
+    "\n",
+    "The 26 unit difference in log evidence between the two models translates to a _huge_ odds ratio in favour of the quadratic model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "Incidentally those data did not come from *either* a linear or quadratic model..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "The same Jeffreys scale used to interpret the information criteria can be used to interpret evidence ratios:\n",
+    "\n",
+    "| $R$    | Strength of Evidence for Model 2 |\n",
+    "|--------|:------------------------:|\n",
+    "|  < 1   | Negative |\n",
+    "|  1-3   | Barely worth mentioning |\n",
+    "|  3-10  | Substantial |\n",
+    "| 10-30  | Strong |\n",
+    "| 30-100 | Very strong |\n",
+    "|   >100 | Decisive |\n",
+    "\n",
+    "> The Bayesian Information Criterion (BIC) is an approximation of $R$ (assuming $N$ datapoints greatly outnumber $k$ parameters, and the priors are uninformative)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "**Calculating the evidence**\n",
+    "\n",
+    "Estimates directly calculated from Markov chains produced for parameter inference are generally not reliable.\n",
+    "\n",
+    "Good methods include nested sampling (e.g. [MultiNest](https://github.com/JohannesBuchner/PyMultiNest)) and parallel tempering (e.g. [emcee](http://dan.iel.fm/emcee/current/))."
```

Comment:
I heard that the parallel tempering implementation in emcee had some issues and they removed it as a result, so maybe take out the link.

Reply:
It's still listed in the docs, as "added in v1.2" - so seems to be active still.